.github/template-files/config.yml

```
@@ -0,0 +1,56 @@
+conda/governance:
+  # [required] community files
+  - CODE_OF_CONDUCT.md
+
+conda/infrastructure:
+  # [required] general workflows
+  - .github/workflows/cla.yml
+  - .github/workflows/update.yml
+
+  # [optional] to include repo in https://github.com/orgs/conda/projects/2
+  # - .github/workflows/issues.yml
+  # - .github/workflows/labels.yml
+  # - .github/workflows/project.yml
+
+  # [optional] stale bot workflows
+  - .github/workflows/stale.yml
+  - .github/workflows/lock.yml
+
+  # [optional] general processes for the conda org
+  - src: templates/HOW_WE_USE_GITHUB.md
+    dst: HOW_WE_USE_GITHUB.md
```

Adjusting to match CRM
```suggestion
  # [optional] general processes for the conda org
  # - src: templates/HOW_WE_USE_GITHUB.md
  #   dst: HOW_WE_USE_GITHUB.md
```